integration_tests/environment.py

```diff
@@ -70,6 +70,7 @@ def before_scenario(context, scenario):
     context.simdir = "./gsy_e-simulation/integration_tests/"
     os.makedirs(context.simdir, exist_ok=True)
     context.resource_manager = contextlib.ExitStack()
+    GlobalConfig.FUTURE_MARKET_DURATION_HOURS = 0
```

Comment:
Shouldn't this be reverted in the `after_scenario`?

Reply:
Indeed, however after your comment I thought that since we want this setting to be applied globally it should be only in before_all (in order for this to be valid for all the integration tests, essentially to be applied once). I'll move it, thanks!